test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist_inductor.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import sys
+from packaging.version import Version
+from packaging.specifiers import SpecifierSet
+
+import numpy as np
+import pytest
+import sagemaker
+from sagemaker.pytorch import PyTorchModel
+
+import boto3
+from datetime import datetime, timedelta
+import time
+import json
+import logging
+
+from ...integration import model_cpu_dir
+from ...integration.sagemaker.timeout import timeout_and_delete_endpoint
+from .... import invoke_pytorch_helper_function
+
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+
+
```

Comment:
Please add coverage for Graviton instances c6g and c7g

Reply:
sagemaker test log (c6g, c7g): https://tiny.amazon.com/10hxj1pvt/IsenLink

Reply:
c6g in SM test will be triggered by `test_mnist_distributed_cpu_inductor` test